src/UI/Components/ConsentScreenComp/OtherConsentComponents.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Text, Image} from 'react-native';
+import styles from '../../Screens/ConsentScreens/Style/OtherConsentStyles';
+import { OtherConsentsConstants } from '../../Screens/ConsentScreens/Constants';
+
+const constants = OtherConsentsConstants();
+
+/**
+ * LocationConsent component
+ */
+const LocationConsent = () => {
```

Comment:
Why two components for the same purpose ? make them one and take parameters in props

Reply:
Done